deploy/helm/flightctl/templates/flightctl-alertmanager-proxy-deployment.yaml

```diff
@@ -95,6 +95,7 @@ spec:
             items:
               - key: ca.crt
                 path: ca.crt
+                optional: true
```

Comment:
There's a warning when installing the helm chart,
```I0730 11:29:27.365397   17955 warnings.go:110] "Warning: unknown field \"spec.template.spec.volumes[0].secret.items[0].optional\""```

Seems like `optional` should be outside `items`